crates/router/src/core/payouts/helpers.rs

```diff
@@ -207,8 +207,8 @@ pub fn should_create_connector_transfer_method(
             .connector_mandate_details
             .clone()
             .map(|details| {
-                details.parse_value::<diesel_models::PaymentsMandateReference>(
-                    "connector_mandate_details",
+                details.parse_value::<diesel_models::CommonMandateReference>(
```

Comment:
Instead of doing this, we will use the implementation defined in PaymentMethod for performing the required steps and getting a CommonMandateReference

Reply:
ok